client/src/components/organisms/AddMemberModal/index.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useState } from "react";
+import AddPeopleButton from "~/components/atoms/AddPeopleButton";
+import DialogBox from "~/components/templates/DialogBox";
+import MemberFilter from "../MemberFilter";
+import { teamData } from "~/shared/jsons/teamData";
+import { ChevronDown, Search } from "react-feather";
+import PeopleList from "~/components/molecules/PeopeList";
+import MemberList from "~/components/molecules/MemberList";
+
```

Comment:
Separate and group imports
```suggestion
import React, { useState } from "react";
import { ChevronDown, Search } from "react-feather";

import AddPeopleButton from "~/components/atoms/AddPeopleButton";
import DialogBox from "~/components/templates/DialogBox";
import MemberFilter from "../MemberFilter";
import { teamData } from "~/shared/jsons/teamData";
import PeopleList from "~/components/molecules/PeopeList";
import MemberList from "~/components/molecules/MemberList";
```